runtime/jextractnatives/CMakeLists.txt

```diff
@@ -39,7 +39,6 @@ target_link_libraries(j9jextract
 )
 
 omr_add_exports(j9jextract
-	Java_com_ibm_jvm_j9_dump_extract_Main_doCommand
```

Comment:
Line 26 must be deleted as it refers to a file that is being deleted.
While you're here, please also delete (mostly blank) line 28.

Reply:
Shall I also remove line 34 as it is mostly blank too?

Reply:
Yes, I think that's a good idea.